packages/gatsby-theme-store/src/gatsby-browser.tsx

```diff
@@ -1,25 +1,26 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-/* eslint-disable @typescript-eslint/no-require-imports */
+// eslint-disable-next-line spaced-comment
+/// <reference types="react-dom/experimental" />
+
```

Comment:
this is needed?

Reply:
Yes, so the tsc compiler infers correctly the import
```
import { unstable_createRoot as createRoot } from 'react-dom'
```